app/controllers/users/registrations_controller.rb

```diff
@@ -44,9 +44,15 @@ class Users::RegistrationsController < Devise::RegistrationsController
   # end
 
   # If you have extra params to permit, append them to the sanitizer.
-  # def configure_account_update_params
-  #   devise_parameter_sanitizer.permit(:account_update, keys: [:attribute])
-  # end
+  def configure_account_update_params
+    devise_parameter_sanitizer.permit(:account_update) do |user|
+      user.permit(:email, :password, :password_confirmation, :current_password, {:profile => [:first_name, :last_name, :gender, :birth_date, :school, :street_name, :zip_code, :city, :state, :country, :graduates_school_in]})
```

Comment:
This line always triggers `undefined method `concat' for #<Proc:0x007f65c4439870>`  after I try to save changes. I tried several different things and this is where I'm stuck.